src/main/java/com/band/api/repository/BandMemberDao.java

```diff
@@ -0,0 +1,35 @@
+package com.band.api.repository;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import java.io.Serializable;
+import java.util.Date;
+
+@Entity
+@Builder
+@Getter
+@Table(name = "band_member", catalog = "band")
+public class BandMemberDao implements Serializable {
```

Comment:
Should all of the DAO classes implement Serializable or why only this one?

Reply:
Only tables with composite primary keys need to implement Serializable.